src/selection.js

```diff
@@ -94,26 +153,35 @@ function Selection(nativeSelection) {
 // TODO: refactor as needed.
 function getSelectionCoords() {
   var sel = document.selection, range, rects, rect;
-  var x = 0, y = 0;
+  var x = 0, y = 0, width = 0;
```

Comment:
should these be set to like, -99999 or null/undefined coz they might conceivably be zero under normal circumstances (& code checks they're 0 to detect if they were unset)